### PR TITLE
< AnalyticsSummaryCard /> - Expose chart animation duration

### PR DESCRIPTION
**Pull request by adamg-wix**
### 🔦 Summary
Exposing Sparkline animation duration in AnalyticsSummaryCard component , it is needed to sometimes disable the animation entirely (i.e empty states) therefore i am exposing this so that i can be set to 0 or maybe in other cases have a slower animation.

The default in the component is set to the default in the sparkline component = 300.

### ✅ Checklist

- 📚 Change is documented
  - [x] API description

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6987/

**Comment by mykas** [comment]
Hey, thanks for the contribution! can you share your use case where you need to disable animations?

**Comment by adamg-wix** [comment]
> Hey, thanks for the contribution! can you share your use case where you need to disable animations?

Screen Shot 2021-03-31 at 14 21 37

This is an empty state of the analytics widget, as you can see this screen has tabs and switching between them causes the animation to re-animate each time, we want to chart to be static.

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.js [hunk 1]
```diff
@@ -248,6 +250,9 @@ AnalyticsSummaryCard.propTypes = {
   /** Sets the color of the chart  */
   chartColorHex: PropTypes.string,
 
+  /** Chart animation duration  */
+  chartAnimationDuration: PropTypes.number,
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.js:254]: Missing d.ts files update.

Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.js:254]: added

**Review by mykas — APPROVED** [review]
Looks good! Thank you

**Pull request merged**